hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/AppContext.java

```diff
@@ -37,34 +36,54 @@ interface Gossip {
          * A {@link Gossip} that throws an exception indicating it should never have been used; for example,
          * if the client code was running in a standalone mode.
          */
-        Gossip UNAVAILABLE_GOSSIP = body -> {
-            throw new IllegalArgumentException("" + FAIL_INVALID);
+        Gossip UNAVAILABLE_GOSSIP = new Gossip() {
+            @Override
+            public void submit(@NonNull final TransactionBody body) {
+                throw new IllegalStateException("Gossip is not available!");
+            }
+
+            @Override
+            public Signature sign(final byte[] ledgerId) {
+                throw new IllegalStateException("Gossip is not available!");
+            }
         };
 
         /**
          * Attempts to submit the given transaction to the network.
+         *
          * @param body the transaction to submit
-         * @throws IllegalStateException if the network is not active; the client should retry later
+         * @throws IllegalStateException    if the network is not active; the client should retry later
          * @throws IllegalArgumentException if body is invalid; so the client can retry immediately with a
-         * different transaction id if the exception's message is {@link ResponseCodeEnum#DUPLICATE_TRANSACTION}
+         *                                  different transaction id if the exception's message is {@link ResponseCodeEnum#DUPLICATE_TRANSACTION}
          */
         void submit(@NonNull TransactionBody body);
+
+        /**
+         * Signs the given byte with the node's RSA key and returns the signature.
```

Comment:
Nit (can wait for future PR):
```suggestion
         * Signs the given bytes with the node's RSA key and returns the signature.
```

Reply:
Fixed. Thanks!